tests/integration/kubernetes/run_kubernetes_nv_tests.sh

```diff
@@ -11,6 +11,16 @@ kubernetes_dir=$(dirname "$(readlink -f "$0")")
 # shellcheck disable=SC1091 # import based on variable
 source "${kubernetes_dir}/../../common.bash"
 
+# Enable NVRC trace logging for NVIDIA GPU runtime
+enable_nvrc_trace() {
```

Comment:
We need this for the SNP and TDX runtimes as well, I would generalize this if the runtime has `nvidia-gpu` in its name enable NVRC debug